Pokemon/Pokemon/PokemonMasterViewController.h

```diff
@@ -8,8 +8,9 @@
 
 #import <UIKit/UIKit.h>
 
-@interface ViewController : UIViewController
+@interface PokemonMasterViewController : UIViewController
 
+@property (strong, nonatomic) IBOutletCollection(UIImageView ) NSArray *pokemonImageViews;
```

Comment:
It looks like you're trying to display a collection of objects—a `UITableView` or `UICollectionView` would make your implementation a lot more flexible. As things are now, you'd have to modify your interface any time you want to add or remove a Pokemon from the UI.